vnext/Desktop/ABI/TestController.h

```diff
@@ -11,7 +11,9 @@ struct TestController {
 
   static Microsoft::ReactNative::IJSValueReader CreateDynamicReader(Microsoft::ReactNative::IJSValueWriter writer);
   static Microsoft::ReactNative::IJSValueWriter CreateDynamicWriter();
-  static Microsoft::ReactNative::IReactContext CreateTestContext();
+  static Microsoft::ReactNative::IReactContext CreateContext(
```

Comment:
Out of curiosity, why is this "Test" code located in the Desktop project?

Reply:
A lot of the ABI functionality we want to test is tied to instance management which we haven't hooked up to the C++/WinRT ABI yet. The TestController is an artificial "entry point" that allows us to test instance dependencies, thus enabling an incremental rather than an all-or-nothing workflow.